### PR TITLE
Import CrUX data into hosts_min_ranks table

### PR DESCRIPTION
**Pull request by jgraham**
This is a direct view of the specific data we want: global minimum rank, local minimum rank, and sightline minimum rank. The requirement for a subquery to select the latest CrUX import makes it hard to write as a materialized view, but since we only need to update it when the CrUX data itself updates, adding it to the ETL makes sense.

Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is
  referenced, the pull request should include the bug number in the title)
- [ ] Scan the PR and verify that no changes (particularly to
  `.circleci/config.yml`) will cause environment variables (particularly
  credentials) to be exposed in test logs
- [ ] Ensure the container image will be using permissions granted to
  [telemetry-airflow](https://github.com/mozilla/telemetry-airflow/)
  responsibly.

### CODE REVIEW

**Comment by jgraham** [comment]
CC @denschub

File: jobs/webcompat-kb/webcompat_kb/crux.py [hunk 1]
```diff
@@ -116,6 +116,47 @@ def update_sightline_data(client: bigquery.Client, config: Config, date: int) ->
...
+        insert_str = ""
+
+    query = f"""
+{insert_str}
+SELECT
+  {date} as yyyymm,
+  NET.HOST(origin) AS host,
+  MIN(
+  IF
+    (origin_ranks.country_code = "global", origin_ranks.rank, NULL)) AS global_rank,
```
Comment (denschub) [inline jobs/webcompat-kb/webcompat_kb/crux.py:132]: Can we put these all in one line, as in

```
MIN(IF(origin_ranks.country_code = "global", origin_ranks.rank, NULL)) AS global_rank,
```
?

That would make this a bit easier to understand

**Review by denschub — APPROVED** [review]
I have a minor style nit, but no hard feelings either way.

File: jobs/webcompat-kb/webcompat_kb/crux.py (hunk 1 above)
Comment (jgraham) [reply jobs/webcompat-kb/webcompat_kb/crux.py:132]: I also kind of hate it, but that's what the Big Query formatter produces, so I'm tempted to leave it.

Comment (denschub) [reply jobs/webcompat-kb/webcompat_kb/crux.py:132]: If an autoformatter did that, I'll concede my point.

**Pull request merged**